app/src/main/java/com/standuptracker/database/NoteDTO.kt

```diff
@@ -0,0 +1,17 @@
+package com.standuptracker.database
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import java.sql.Date
+
+
+@Entity(tableName="note_table")
+class Note(
+    @PrimaryKey val noteID: Int,
+    @ColumnInfo(name="note_content") val noteContent:String,
+    @ColumnInfo(name="created_by") val createdBy: Int,
```

Comment:
change column name to "userID" and add a foreign key constraint back to the user table